src/components/radio-group/radio-group.tsx

```diff
@@ -124,6 +119,7 @@ export class RadioGroup extends React.Component<RadioGroupProps> {
                 name={this.name}
                 className="option"
                 tabIndex={this.getChildTabIndex(index, this.isGroupChecked)}
+                autoFocus={this.isGroupChecked ? this.checkedArray[index].checked : index === 0}
```

Comment:
I feel this should be abstracted or simplified. A trinary operator that ends with another condition check could be confusing.